controllers/usersTalentsController.js

```diff
@@ -1,30 +1,24 @@
 var db = require('../models');
 
-// GET '/api/users/:userId/talents'
```

Comment:
You don't need this route – you get all of a user's talents when you get a user, because the Talent schema is embedded in the User model.